mephisto/abstractions/blueprints/parlai_chat/parlai_chat_blueprint.py

```diff
@@ -285,17 +285,23 @@ def get_frontend_args(self) -> Dict[str, Any]:
         Specifies what options within a task_config should be fowarded
         to the client for use by the task's frontend
         """
-        # TODO move frontend args in
-        frontend_task_config = {
-            "task_description": self.full_task_description,
-            "preview_html": self.full_preview_description,
-            "frame_height": 0,
-            "chat_title": self.args.task.task_title,
-            "has_preview": self.args.blueprint.get("preview_source", None) is not None,
-            "block_mobile": True,
-            "frontend_task_opts": self.shared_state.frontend_task_opts,
-        }
-        frontend_task_config.update(super().get_frontend_args())
+        # Start with standard task configuration arguments
+        frontend_task_config = super().get_frontend_args()
+        # Add ParlAI standards
+        frontend_task_config.update(
+            {
+                "task_description": self.full_task_description,
+                "preview_html": self.full_preview_description,
+                "frame_height": 650,
+                "chat_title": self.args.task.task_title,
+                "has_preview": self.args.blueprint.get("preview_source", None)
```

Comment:
this is an intentional change right? whereas before it would enforce a boolean this may let non-boolean values flow through? could this cause breakages?

Reply:
Ah it's on the next line - `black` formatting decided to make this less clear unfortunately

Reply:
ah lol I see it now... interesting choice `black`...